### DISCUSSION TITLE
Service Filter/rules causing conflict with each other

### DISCUSSION THREAD

**Discussion by ishanjain28** [AdguardTeam/AdGuardHome#3751] [General]
Hi there,

Adguard home has a Blocked services feature which is essentially a list of hard coded filters that are applied when you choose to block a service.

Over the last few weeks(or maybe even before that, I just noticed it recently), Instagram has started using Facebook's `fbcdn.net` service. 

The iOS/Android apps now send requests to, 

```
instagram.flko4-1.fna.fbcdn.net
instagram.fbek1-1.fna.fbcdn.net
instagram.fdel41-1.fna.fbcdn.net
```

and a lot of similar looking domains. 
The Facebook block service has a filter, `||fbcdn.net^` which blocks access to the domains used by Instagram as well. 

This means, If you block facebook, You will also unintentionally end up blocking Instagram as well and this is what happened in my network. Instagram was not accessible for anyone for the last few weeks and they only told me about it today. :| 

My question is that, Is this something within the scope of the Blocked services feature? Is this project tracking breaking changes like this in this feature?

**Comment by ainar-g** [comment]
I think that in general we accept such feedback in the recurrent issues like #2644, but I cannot find one for blocked services.

@ameshkov, perhaps we should make a recurrent issue about updating data for blocked services in each release?  Or would you prefer to have separate issues for each case?

**Reply by ameshkov** [reply]
@ainar-g I'd prefer a separate issue for every case tbh.

**Comment by ishanjain28** [comment]
Alright. I'll create a issue for this.
